packages/spectacular/src/lib/feature-testing/configuration/with-initial-feature-navigation.ts

```diff
@@ -0,0 +1,43 @@
+import { initialFeatureNavigationInitializer } from '../navigation/initial-feature-navigation.initializer';
+import { makeSpectacularFeatureTestingFeature } from './make-spectacular-feature-testing-feature';
+import type { provideSpectacularFeatureTesting } from './provide-spectacular-feature-testing';
```

Comment:
nitpick: a nice warning from the linter that provideSpectacularFeatureTesting is not being used

Reply:
It's used by TSDoc `@link` annotations.